src/vm/typeparse.cpp

```diff
@@ -1321,7 +1325,8 @@ TypeHandle TypeName::GetTypeFromAsm()
     if (th.IsNull() && bThrowIfNotFound)
     {
         StackSString buf;
-        LPCWSTR wszName = ToString(&buf)->GetUnicode();
+        SString* bufString = ToString(&buf);
```

Comment:
Was this refactoring required for some reason, or is it just a cosmetic change?

Reply:
It is a leftover, I will revert it. I first added null checks in all usages, and this refactoring prevented calling ToString() twice. In next compile, the null check was reverted but code refactoring remained.